tinygrad/device.py

```diff
@@ -249,12 +249,9 @@ def get_linearizer(self, ast:LazyOp) -> Linearizer:
     k = Linearizer(ast, self.compiler.linearizer_opts)
     k.required_optimizations()
     if not NOOPT:
-      if not (used_tensor_cores:=k.apply_tensor_cores(getenv("TC", 1))): k.hand_coded_optimizations()
+      k.hand_coded_optimizations()
       if BEAM >= 1:
-        lins = [(("tc" if used_tensor_cores else "hc"), k)]
-        if used_tensor_cores:
-          lins.append(("hc", Linearizer(ast, self.compiler.linearizer_opts)))
-          lins[-1][1].hand_coded_optimizations()
+        lins = [("hc", k)]
```

Comment:
I don't like that this doesn't try the hand coded and tensor cores separate anymore? Why did this have to change?